### PR TITLE
Add execution node field to job details panel

### PR DESCRIPTION
**Pull request by marshmalien**
##### SUMMARY
Link: https://github.com/ansible/awx/issues/2330
* Add the node that executed the job to the job output details panel.

screen shot 2019-01-28 at 1 45 05 pm

##### ISSUE TYPE
 - Enhancement Pull Request

##### COMPONENT NAME
 - UI

### CODE REVIEW

File: awx/ui/client/features/output/details.component.js [hunk 1]
```diff
@@ -481,6 +481,19 @@ function getLimitDetails () {
     return { label, value };
 }
 
+function getExecutionNodeDetails () {
+    const executionNode = resource.model.get('execution_node');
+
+    if (!executionNode) {
```
Comment (jakemcdermott) [inline awx/ui/client/features/output/details.component.js:487]: 👍

File: awx/ui/client/features/output/output.strings.js [hunk 2]
```diff
@@ -51,6 +51,7 @@ function OutputStrings (BaseString) {
     ns.labels = {
         CREDENTIAL: t.s('Credential'),
         ENVIRONMENT: t.s('Environment'),
+        EXECUTION_NODE: t.s('Execution Node'),
```
Comment (jakemcdermott) [inline awx/ui/client/features/output/output.strings.js:54]: 👍

**Review by jakemcdermott — APPROVED** [review]

File: awx/ui/client/features/output/details.component.js [hunk 3]
```diff
@@ -481,6 +481,19 @@ function getLimitDetails () {
...
+        return null;
+    }
+
+    const label = strings.get('labels.EXECUTION_NODE');
+    const value = $filter('sanitize')(executionNode);
```
Comment (jakemcdermott) [inline awx/ui/client/features/output/details.component.js:492]: 👍 https://github.com/ansible/awx/pull/3097

**Review by elyezer — APPROVED** [review]
I played with these changes locally and they are working fine.

File: awx/ui/client/features/output/details.component.js [hunk 4]
```diff
@@ -782,12 +796,21 @@ function JobDetailsController (
...
 
-        unsubscribe = subscribe(({ status, started, finished, scm, inventoryScm, environment }) => {
+        unsubscribe = subscribe(({
+            status,
+            started,
+            finished,
+            scm,
+            inventoryScm,
+            environment,
+            executionNode
```
Comment (jakemcdermott) [inline awx/ui/client/features/output/details.component.js:806]: 👍

File: awx/ui/client/features/output/status.service.js [hunk 5]
```diff
@@ -321,6 +327,7 @@ function JobStatusService (moment, message) {
                 this.setStarted(model.get('started'));
                 this.setJobStatus(model.get('status'));
                 this.setEnvironment(model.get('custom_virtualenv'));
+                this.setExecutionNode(model.get('execution_node'));
```
Comment (jakemcdermott) [inline awx/ui/client/features/output/status.service.js:330]: 👍

**Review by jakemcdermott — APPROVED** [review]

**Pull request merged**